code/modules/antagonists/cult/runes.dm

```diff
@@ -148,7 +148,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 		else if(istype(M, /obj/item/toy/plush/narplush))
 			var/obj/item/toy/plush/narplush/P = M
 			P.visible_message("<span class='cult italic'>[P] squeaks loudly!</span>")
-	if(!src.density) //yogs: barrier runes play cooldown animation properly
```

Comment:
Why are you removing this?

Reply:
don't need it since we aren't mirroring TG

Reply:
It's helpful for when we rebase again in the future. Keeping them doesn't hurt